### PR TITLE
Add support for ".docker" directories

### PR DESCRIPTION
**Pull request by CarlRosell**
I was just browsing my home directory and thought it would be fun to have the `.docker` directory visually show that it has to do with Docker. 🙂

### CODE REVIEW

File: config.cson [hunk 1]
```diff
@@ -93,6 +93,11 @@ directoryIcons:
 		icon: "circleci"
 		match: /^\.circleci$/
 
+  Docker:
+		icon: "docker"
+    match: /^\.docker$/
```
Comment (Alhadis) [inline config.cson]: Use tabs. Not spaces.

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 - **New icons:** Edge (`.edge`), Firebase (`.firebaserc`, `firebase.json`), GLSL (`.glsl`, `.glslv`, `.gsh`, `.gshader`), Vertex/Fragment Shader (`.fp`, `.frag`, `.frg`, `.fsh`, `.fshader`, `.vert`, `.vertex`, `.vrx`, `.vsh`), SketchUp LayOut (`.layout`), SketchUp Make (`.skp`), SketchUp Style Builder (`.style`), WebGL (`.webgl`)
 - **Support:** 3D Asset (`.comp`, `.geo{m,metry}`, `.tesc`, `.tese`), ChartJS (`chart.{bundle,min}.js`), NPM (`package-lock.json`)
+- **Support:** Docker (`.docker` directories)
```
Comment (Alhadis) [inline CHANGELOG.md]: Start a new section in the change-log, please. This isn't part of v2.1.5, which has already been released:

~~~markdown
[Unpublished]
---------------------
### Added
- **Support:** Docker (`.docker` directories)
~~~

**Comment by Alhadis** [comment]
Hi @CarlRosell,

Are `.docker` directories common? I'm not familiar with Docker, I'm afraid.

**Comment by CarlRosell** [comment]
Hi!
Sorry, my editor config plugin was turned off.
I believe that `.docker` is the default for Docker configurations. I have them on both of my computers, and they are mentioned around the web 🙂

**Comment by CarlRosell** [comment]
Thanks 👍

**Comment by Alhadis** [comment]
My pleasure!

**Pull request merged**